producer-c/producer-cloudwatch-integ/KvsProducerSampleCloudwatch.cpp

```diff
@@ -162,13 +176,16 @@ INT32 main(INT32 argc, CHAR *argv[])
             frame.flags = frameIndex % DEFAULT_KEY_FRAME_INTERVAL == 0 ? FRAME_FLAG_KEY_FRAME : FRAME_FLAG_NONE;
 
             createCanaryFrameData(&frame);
-
             if (frame.flags == FRAME_FLAG_KEY_FRAME) {
                 if (lastKeyFrameTimestamp != 0) {
                     canaryStreamRecordFragmentEndSendTime(pCanaryStreamCallbacks, lastKeyFrameTimestamp, frame.presentationTs);
                     CHK_STATUS(computeStreamMetricsFromCanary(streamHandle, pCanaryStreamCallbacks));
                     CHK_STATUS(computeClientMetricsFromCanary(clientHandle, pCanaryStreamCallbacks));
                     currentMemoryAllocation(pCanaryStreamCallbacks);
+                    if(GETTIME() > currentTime + (60 * HUNDREDS_OF_NANOS_IN_A_SECOND)) {
```

Comment:
Minor. You can store the value of GETTIME() here and assign to the currentTIme

Reply:
Is there an advantage to doing that?

Reply:
Not calling it twice. Most platforms will have a minimal impact. Certainly, for this you can ignore